accepted/future-releases/0546-patterns/feature-specification.md

```diff
@@ -782,8 +798,8 @@ In both record patterns and object patterns, a field subpattern's name may be
 elided when it can be inferred from the field's value subpattern. The inferred
 field name for a pattern `p`, if one exists, is defined as:
 
-*   If `p` is a variable pattern which binds a variable `v`, and `v` is not `_`,
-    then the inferred name is `v`.
+*   If `p` is a variable or identifier pattern which binds or assigns a variable
```

Comment:
Perhaps
```suggestion
*   If `p` is a variable or identifier pattern whose identifier is
```
There is no doubt about which identifier we are referring to because both the variable pattern syntax and the identifier pattern syntax has exactly one `<identifier>`. With this change we'd avoid the semantic knot "but if `v` is `_` then it doesn't actually bind or assign any variables". This knot could create the idea in some readers' minds that `_` _could_ bind a variable named `_` in some mysterious situations.

Reply:
Good idea, done.